roles/openshift_metrics/tasks/generate_hawkular_certificates.yaml

```diff
@@ -47,22 +32,18 @@
   - hawkular-metrics.pwd
   - hawkular-metrics.htpasswd
 
-- include: import_jks_certs.yaml
-
 - name: read files for the hawkular-metrics secret
   shell: >
     printf '%s: ' '{{ item }}'
     && base64 --wrap 0 '{{ mktemp.stdout }}/{{ item }}'
   register: hawkular_secrets
   with_items:
   - ca.crt
-  - hawkular-metrics.crt
-  - hawkular-metrics.keystore
-  - hawkular-metrics-keystore.pwd
-  - hawkular-metrics.truststore
-  - hawkular-metrics-truststore.pwd
   - hawkular-metrics.pwd
   - hawkular-metrics.htpasswd
+  - hawkular-metrics.crt
+  - hawkular-metrics.key
+  - hawkular-metrics.pem
```

Comment:
so we're going to use this .pem generated in `setup_certificate` instead of the pkcs12 file essentially?